pem/README.md

```diff
@@ -39,6 +39,8 @@ Tired of manually creating and maintaining your push notification profiles for y
 
 `pem` does all that for, just by running `pem`.
 
+`pem` will check if there is a valid push notification profile, and if not, create a new one. When it does this, it will creates a pem, cer and p12 file. Pem does not cover uploading the file to your server
```

Comment:
"`pem` creates new `.pem`, `.cer`, and `.p12` files to be uploaded to your push server if a valid push notification profile is needed."